### ISSUE TITLE
TSML JSON Encoding Requirements

### ISSUE THREAD

**Issue by phersh2** [opengeospatial/timeseriesML#2]

**Comment by hylkevds** [comment]
It would be nice if the encoding allows referencing the "next" document in a series.
There will be data sets that are too large to transmit in a single document.
It would also be very useful for APIs that do pagination.

**Comment by phersh2** [comment]
Yes. Furthermore, we have TSML 2.0 requirements/suggestions from NASA to split the payload for large data sets (i.e., splitting data payload from metadata, etc).

See  use case c. (SORCE) and d. (SWOT) under the six use cases in [NASA's final report](https://docs.google.com/document/d/1XLDkYktX1r0hlNK8g9Wv7i6I5Hi36r7NG4hXzkMpGEg/edit?usp=sharing)
